packages/lodestar/src/network/reqresp/encodingStrategies/sszSnappy/encode.ts

```diff
@@ -1,21 +1,23 @@
 import varint from "varint";
 import {source} from "stream-to-it";
 import {createCompressStream} from "@chainsafe/snappy-stream";
-import {RequestOrResponseType, RequestOrResponseBody} from "../../types";
+import {RequestOrResponseType, RequestOrLodestarResponseBody} from "../../types";
 import {SszSnappyError, SszSnappyErrorCode} from "./errors";
+import {P2pBlockResponse} from "@chainsafe/lodestar-types";
 
 /**
  * ssz_snappy encoding strategy writer.
  * Yields byte chunks for encoded header and payload as defined in the spec:
+ * @param type: null if we use bytes data without having to serialize
  * ```
  * <encoding-dependent-header> | <encoded-payload>
  * ```
  */
-export async function* writeSszSnappyPayload<T extends RequestOrResponseBody>(
+export async function* writeSszSnappyPayload<T extends RequestOrLodestarResponseBody>(
   body: T,
-  type: RequestOrResponseType
+  type: RequestOrResponseType | null
 ): AsyncGenerator<Buffer> {
-  const serializedBody = serializeSszBody(body, type);
+  const serializedBody = type === null ? (body as P2pBlockResponse).bytes : serializeSszBody(body, type);
```

Comment:
Instead of nulling the type you can define the type as an object that
```ts
const reqRespOutgoingResponseBlockType = {
  serialize(serializedBlockWrapper: SerializedBlockWrapper) {
    return serializedBlockWrapper.bytes
  }
}
```